README.md

```diff
@@ -70,8 +70,28 @@ dependencies {
 ### Using the AWS JDBC Driver for MySQL
 As a drop-in compatible, usage of the AWS JDBC Driver for MySQL, is identical to the [MySQL-Connector-J JDBC driver](https://github.com/mysql/mysql-connector-j). The sections below highlight usage specific to failover.
 
+#### :warning: Warnings About Proper Usage of the AWS JDBC Driver for MySQL
```

Comment:
Does it make sense to move this entire section at the end of chapter, right before "Developement"?
Maybe add some dark grey background for entire warning section?

Reply:
I'm fine with doing that, as long as you think it won't be missed if its lower down in the doc. As far as I can tell there isn't a way to create a different colored background (eg box) for GH markdown, but I'll look into other ways to emphasize the section